met-web/src/components/engagement/new/create/engagmentCreateAction.tsx

```diff
@@ -22,6 +23,13 @@ export const engagementCreateAction: ActionFunction = async ({ request }) => {
     } catch (e) {
         console.error('Error updating engagement slug', e);
     }
+    formData
+        .get('users')
+        ?.toString()
+        .split(',')
+        .forEach(async (user_id) => {
```

Comment:
I don't think async/await works with forEach loops. https://stackoverflow.com/questions/37576685/using-async-await-with-a-foreach-loop

If you want to wait for the calls to finish before returning, you could use promise.all, for example